data/json/items/gunmod/accessories.json

```diff
@@ -68,5 +68,22 @@
     "loudness_modifier": -8,
     "handling_modifier": 1,
     "min_skills": [ [ "weapon", 2 ] ]
+  },
+  {
+    "id": "belt_clip",
+    "type": "GUNMOD",
+    "name": "belt clip",
+    "description": "This is a belt clip that attaches to the grip or slide of a pistol so as to facilitate 'mexican carry', the practice of carrying without a holster.  It does not offer any protection for the trigger, so users are strongly advised to carry with the chamber empty or select a firearm with a very heavy trigger pull.",
+    "weight": 40,
+    "volume": 1,
+    "integral_volume": 0,
+    "price": 2000,
+    "material": "steel",
+    "symbol": ":",
+    "color": "light_gray",
+    "armor_data": { "covers": [ "LEG_EITHER" ] },
+    "location": "accessories",
+    "mod_targets": [ "rugerlcp", "kp32", "kp3at", "kpf9", "cop_38", "moss_brownie", "moss_brownie" ],
```

Comment:
You specified "moss_brownie" twice here, probably not necessary.

Reply:
Thanks, missed that.